hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/SCMPipelineManager.java

```diff
@@ -523,4 +543,21 @@ protected File getPipelineDBPath(Configuration configuration) {
     File metaDir = ServerUtils.getScmDbDir(configuration);
     return new File(metaDir, SCM_PIPELINE_DB);
   }
+
+  protected ReadWriteLock getLock() {
```

Comment:
Instead of a get on a lock object, it would make sense to make the reference accessible.

Reply:
That will lead to findbug violation. Do you see any issues using a getter?

Reply:
I see, ok let's ignore this comment.